subworkflows/local/prepare_genome/main.nf

```diff
@@ -115,6 +118,7 @@ workflow PREPARE_GENOME {
     emit:
     bwa                   = BWAMEM1_INDEX.out.index.map{ meta, index -> [index] }.collect()       // path: bwa/*
     bwamem2               = BWAMEM2_INDEX.out.index.map{ meta, index -> [index] }.collect()       // path: bwamem2/*
+    sentieon              = SENTIEON_BWAINDEX.out.index.map{ meta, index -> [index] }.collect()   // path: bwa/*
```

Comment:
can we merge bwa and sentieon?
only one index should be there anyway

Reply:
Yeah, so we don't include sentieon-bwaindex but just use the already present BWAMEM1_INDEX for sentieon-bwamem?

Reply:
yes, and in cases were the index has to be created by the pipeline, then you mix both indice computations